examples/tutorial_groups.ipynb

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
Should this notebook be removed?

Reply:
Not sure now... is this new one replacing it?